gavel_ci/ui/jobserv.py

```diff
@@ -1,15 +1,27 @@
 # Copyright (C) 2018 Open Source Foundries
 # Author: Andy Doan <andy@opensourcefoundries.com>
-import requests
+import logging
 
+import requests
 from flask import (
-    Blueprint, abort, flash, jsonify, make_response, redirect, render_template,
-    request, url_for
+    abort,
+    Blueprint,
+    flash,
+    jsonify,
+    make_response,
+    redirect,
+    render_template,
+    request,
+    Response,
+    url_for,
 )
 from flask_login import current_user, fresh_login_required
 
 from gavel_ci.settings import JOBSERV_URL
 
+
+logger = logging.getLogger(__name__)
```

Comment:
looks like an unused import now?

Reply:
eeeesh I forgot to run flake8 -- ran and fixed, thank you kind sir.